arlo_server/base.py

```diff
@@ -1,4 +1,5 @@
 from flask import Flask
-from config import STATIC_FOLDER
+from config import STATIC_FOLDER, SESSION_SECRET
 
 app = Flask(__name__, static_folder=STATIC_FOLDER)
+app.secret_key = SESSION_SECRET
```

Comment:
does this work? It's not `app.config['SECRET_KEY']` ?

Reply:
They're equivalent, apparently: https://flask.palletsprojects.com/en/1.1.x/api/#flask.Flask.secret_key.